keyboards/zlabkeeb/vrynboard/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2024 zlabkeeb (zlabkeeb@gmail.com)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define OLED_TIMEOUT 180000
+#define I2C1_SDA_PIN GP26
+#define I2C1_SCL_PIN GP27
+#define I2C_DRIVER I2CD1
+#define DYNAMIC_KEYMAP_MACRO_COUNT 50
```

Comment:
Just a heads up, only 32 macros are supported in firmware (via keycodes).  Any more than that ...

Reply:
Alright, thank you. will make the changes right away.

Reply:
Just to be clear, it is possible to support more.  However, the keycodes only exist for the first 32.  And the VIA app doesn't support any more than it's normal number (which I think is 16).

Reply:
Thanks @drashna  😂